### PR TITLE
Remove solana donations from calculation

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #47

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@divine-comedian  Please review it but dont merge it, I will merge it myself to monitor the server to make sure it's working after merging

**Comment by divine-comedian** [comment]
I snuck in one extra small change that Ashley requested - I forgot to add the download options for calculate-updated

**Review by divine-comedian — APPROVED** [review]

**Pull request merged**
